docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -647,6 +647,13 @@ contain certificates, CRLs, and much more. PKCS7 files commonly have a ``p7b``,
             :class:`~cryptography.hazmat.primitives.hashes.SHA384`, or
             :class:`~cryptography.hazmat.primitives.hashes.SHA512`.
 
+    .. method:: add_certificate(certificate)
+
+        Add an additional certificate to the PKCS7 structure. This method may
```

Comment:
Docs ought to describe what the certs are for :-)

Also, does it just put all of these in a ball? They don't need to be assosciated with a particular signer?

Reply:
If PKCS7 allows association it's not in the OpenSSL API. Big ball best ball. Let the verifier sort 'em out I always say.